cmd/config.go

```diff
@@ -61,6 +71,34 @@ const (
 
 var cfg *Config
 
+func (cfg *Config) setProjectWildcardDefaults() {
```

Comment:
I would remove this function entirely and have getters on `Project` and `Wildcard` types. eg:

```go
func (p *Project) FetchPipelineJobStats(cfg) bool {
  if p.FetchPipelineJobStats == nil {
    return cfg.FetchPipelineJobStats
  }
  return *p.FetchPipelineJobStats
}
```
which would allow you to avoid having the duplicate fields on both types.

Reply:
Yep, just pushed a commit which implements this - looks much cleaner. I renamed the methods as you can't use the same name for the method as the pointer but other than that, implemented pretty much as you stated.

Reply:
nice, thanks! 🙇